platform-sdk/swirlds-config-api/src/main/java/com/swirlds/config/api/ConfigurationBuilder.java

```diff
@@ -119,6 +119,16 @@ public interface ConfigurationBuilder {
     @NonNull
     ConfigurationBuilder withConfigDataTypes(@NonNull Class<? extends Record>... types) throws IllegalStateException;
 
+    /**
+     * Sets the value for the config.
```

Comment:
Please change to:
`Adds a property. In the created {@link Configuration} the property will be defined by an internal config source that has the defined ordinal of FOO.`
or similar.

Reply:
done